### PR TITLE
Adb screenshot

### PR DESCRIPTION
**Pull request by xz-dev**
This is the code for screenshots of Android. It requires ADB.

### CODE REVIEW

File: Visual_Studio/wechat_auto_jump.pyproj [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (xiangkaiz) [inline Visual_Studio/wechat_auto_jump.pyproj]: Please do not commit config file of Visual Studio.

**Review by xiangkaiz — CHANGES_REQUESTED** [review]
Please change these.

File: adb.py [hunk 2]
```diff
@@ -1 +1,16 @@
...
+def adb_sc():
+    """ADB截图"""
+    os.chdir('adb')
+    out = os.popen('adb devices').read() # 检查设备状态
+    print(out)
+    if 'device' not in str.split(out):
+        print('请检查设备状态\n')
+        return(0)
+    else:
+        os.popen('adb shell screencap /sdcard/screen.png') # ADB工具截图
```
Comment (xiangkaiz) [inline adb.py]: As I recall, `subprocess` seems to be a better way to create another process.

File: start.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+"""启动文件"""
+import msvcrt
+import adb
+
+out = adb.adb_sc() # 调用ADB模块截图
+if out == 0:
+    print('按任意键结束')
+    msvcrt.getch()
+    exit()
```
Comment (xiangkaiz) [inline start.py]: You should merge the two files into one.

**Review by xiangkaiz — CHANGES_REQUESTED** [review]
Please change these.

File: start.py (hunk 3 above)
Comment (xz-dev) [reply start.py]: I have completed the change.

Comment (xiangkaiz) [reply start.py]: Please translate all Chinese to English. Also, do not use `exit()` and `mscvrt`. You should use main entrance.

**Review by xiangkaiz — CHANGES_REQUESTED** [review]
Translate all Chinese to English, please.

**Review by xiangkaiz — APPROVED** [review]

**Pull request merged**